rest_framework_tracking/mixins.py

```diff
@@ -108,6 +108,31 @@ def _should_log(self, request, response):
         """
         return self.logging_methods == '__all__' or request.method in self.logging_methods
 
+    def _clean_data(self, data):
+        """
+        Clean a dictionary of data of potentially sensitive info before
+        sending to the database.
+        Function based on the "_clean_credentials" function of django
+        (https://github.com/django/django/blob/stable/1.11.x/django/contrib/auth/__init__.py#L50)
+
+        Fields defined by django are by default cleaned with this function
+
+        You can define your own sensitive fields in your view by defining a list
+        eg: sensitive_fields = ['field1', 'field2']
+        """
+        data = dict(data)
+
+        SENSITIVE_FIELDS = ['api', 'token', 'key', 'secret', 'password', 'signature']
+        CLEANED_SUBSTITUTE = '********************'
+
+        if self.sensitive_fields:
+            SENSITIVE_FIELDS += [field.lower() for field in self.sensitive_fields]
+
+        for key in data:
+            if key.lower() in SENSITIVE_FIELDS:
```

Comment:
It seems to be faster (based on what I found there: https://stackoverflow.com/questions/2831212/python-sets-vs-lists).

I'll try no local to make sure it is and do a PR if it is. 

Thanks for the comment